README.md

```diff
@@ -45,9 +45,10 @@ class Foo
 end
 ```
 
-If `#as_json` is called without the `:presenter_action` options, or if the
-presenter class doesn't exist, this delegates to the original model class
-definition of `#as_json`
+If `#as_json` is called without the `:presenter_action` options, or if
+the presenter class doesn't exist, this delegates to
```

Comment:
I think the "if the presenter class doesn't exist" part of this is wrong now. If you include the module in a model, but don't create a presenter for it, it will still call the model's `#as_json` method, because it doesn't have a presenter to call `#as_default_json` on.